internal/dag/status_test.go

```diff
@@ -568,8 +568,8 @@ func TestDAGStatus(t *testing.T) {
 			}},
 			Routes: []contour_api_v1.Route{{
 				Services: []contour_api_v1.Service{{
-					Name: "green",
-					Port: 80,
+					Name: "kuard",
```

Comment:
Why change this?

Reply:
I think the test case has always been wrong:  According to comment, the test case is supposed to test include cycle, but besides that, it has another error as well: it referred to non-existing service "green".  Now that non-existing service reference is not fatal anymore, I noticed this

Reply:
👍